src/excalidraw-app/index.tsx

```diff
@@ -126,17 +141,26 @@ const initializeScene = async (opts?: {
           commitToHistory: true,
         };
       }
-      return false;
+
+      return {
+        ...restore(
```

Comment:
restore will be called in excalidraw so we can remove from here?

Reply:
I did it as a quick hack to normalize the spread value, wouldn't work otherwise. But yes, after we settle the API we'll need to go through the code and clean everything up.